sdk/src/azure/core/az_hfsm.c

```diff
@@ -20,6 +20,8 @@
 #include <azure/core/internal/az_result_internal.h>
 #include <azure/core/internal/az_precondition_internal.h>
 
+#include <azure/core/_az_cfg.h>
```

Comment:
Should this file be named az_hfsm_internal.c?

Reply:
See my [comment](https://github.com/CIPop/azure-sdk-for-c/pull/8#:~:text=I%20am%20not%20certain%2C%20none%20of%20the%20other%20files%20have%20_internal%20as%20a%20suffix%20even%20if%20they%20do%20have%20an%20internal%20.h%20(they%20do%20have%20a%20non%2Dinternal.h%20though).).